<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WLvlDataBaroCorrection</class>
 <widget class="QDialog" name="WLvlDataBaroCorrection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>228</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Log data period configuration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#WLvlDataBaroCorrection {
 background-color:white;
 border-radius: 5px;
}
#label,
#label_2,
#label_4 {
 color: #3E4072;
 font-weight: 800;
}
#buttonBox{
 background-color: #3E4072;
}
#label_Title{
 color: #8F8F8F;
 font-weight: 600;
}
QComboBox,
QDoubleSpinBox {
 background-color:#DFDFDF;
 border-radius: 3px;
 color: black;
}
#pushButton_Confirm,
#pushButton_Cancel {
 border: none;
 border-radius: 8px;
}
#pushButton_Confirm {
 color: white;
 background-color: #3E4072;
}
#pushButton_Confirm:hover {
 color: white;
 background-color: #595D89;
}
#pushButton_Cancel {
 background-color: #D8D8D8;
}
#pushButton_Cancel:hover {
 background-color: #ACACAC;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_Title">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <fontweight>DemiBold</fontweight>
      </font>
     </property>
     <property name="text">
      <string>Baro correction</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Correction type</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_CorrectionType">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Logged Zeros</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baro File</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fixed</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_PartBaroSelected" native="true">
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_BaroSelected">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>2XXXXXXX</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Baro selected</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_PartFixedValue" native="true">
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,0">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Fixed value</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_FixedValue">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_Confirm">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Cancel">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>WLvlDataBaroCorrection</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Confirm</sender>
   <signal>clicked()</signal>
   <receiver>WLvlDataBaroCorrection</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
